### PR TITLE
Added authored range, published and page filter

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | Requires https://github.com/sulu/sulu/pull/3437 https://github.com/sulu/SuluArticleBundle/pull/227 https://github.com/sulu/SuluArticleBundle/pull/228 https://github.com/sulu/SuluArticleBundle/pull/229
| License | MIT

#### What's in this PR?

This PR adds additional filters for Article-List.

* Authored range
* Published state

#### To Do

- [x] Published state
- [x] UI

### CODE REVIEW

**Comment by wachterjohannes** [comment]
@alexander-schranz I have also fixed the UI. you can now review this and the linked PR which are included in this PR

**Comment by wachterjohannes** [comment]
@alexander-schranz you can review this PR now

File: Resources/translations/sulu/backend.fr.xlf [hunk 1]
```diff
@@ -138,6 +138,14 @@
                 <source>sulu_article.list.filter.by-tag</source>
                 <target>Uniquement par tag</target>
```
Comment (alexander-schranz) [inline Resources/translations/sulu/backend.fr.xlf]: missing translation changes

File: Resources/public/js/components/articles/list/authored-selection/main.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        },
+        translations: {
+            title: 'sulu_article.authored',
+            from: 'sulu_article.authored-selection-overlay.from',
+            to: 'sulu_article.authored-selection-overlay.to',
+            reset: 'smart-content.choose-data-source.reset'
+        },
+        templates: {
+            skeleton: [
+                '<div class="grid">',
```
Comment (alexander-schranz) [inline Resources/public/js/components/articles/list/authored-selection/main.js]: move into own file

**Pull request merged**